sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ClassNotInstantiatable.vb

```diff
@@ -99,4 +99,23 @@ Namespace Tests.Diagnostics
         Private Sub New()
         End Sub
     End Class
+
+    ' https//github.com/SonarSource/sonar-dotnet/issues/3329
+    Public Class Repro_3329
```

Comment:
Preserve the original comment here please to explain why is this case compliant (except the Non+FP part)
```suggestion
    Public Class Repro_3329 `Compliant, instance will be created by PInvoke of DllImport
```